content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-nodeable-nodes.md

```diff
@@ -184,61 +184,30 @@ set(FILES
 )
 ```
 
-## Step 5: Reflect the new node {#reflect-the-new-node}
+## Step 5: Register the new node {#register-the-new-node}
 {{< note >}}
 This step is only required once for the first time nodeable node creation.
 {{< /note >}}
 
-The final step is to register and reflect the new node. To do this, you need to modify your Gem's [Gem module](/docs/user-guide/programming/gems/overview/) and [system component](/docs/user-guide/programming/components/system-components/). Use the **StartingPointInput** Gem from the O3DE source as a reference:
-
-1. In your Gem's system component, include the auto-generated registry header file, and invoke `REGISTER_SCRIPTCANVAS_AUTOGEN_NODEABLE` with the sanitized Gem target name.
-    {{< note >}}
-    Use the same auto-generated registry header file that you declared in Step 1 under `AUTOGEN_RULES` in your Gem's `Code/CMakeLists.txt`. In the **StartingPointInput** example, it is `AutoGenNodeableRegistry.generated.h`.
-    {{< /note >}}
-    {{< note >}}
-    The sanitized Gem target name should contain letters and numbers only. In the **StartingPointInput** example, it is `StartingPointInputStatic` which refers to the `StartingPointInput.Static` target.
-    {{< /note >}}
-   
-    For example, in [`StartingPointInputGem.cpp`](https://github.com/o3de/o3de/blob/development/Gems/StartingPointInput/Code/Source/StartingPointInputGem.cpp):
-  
-    ```cpp
-    #include <AutoGenNodeableRegistry.generated.h>
-    ...
-    
-    REGISTER_SCRIPTCANVAS_AUTOGEN_NODEABLE(StartingPointInputStatic);
-    ...
-    ```
+The final step is to register new node. To do this, you need to modify your Gem's [Gem module](/docs/user-guide/programming/gems/overview/) or [system component](/docs/user-guide/programming/components/system-components/). Use the **StartingPointInput** Gem from the O3DE source as a reference:
```

Comment:
`The final step is to register new node` -> `The final step is to register the new node`

Reply:
done